### DISCUSSION TITLE
how do you copy a system

### DISCUSSION THREAD

**Discussion by opoplawski** [cobbler/cobbler#3632] [Q&A]
I'm trying to copy one system to a new one:
```
# cobbler system copy --name f39 --newname f40
exception on server: IP address duplicate found "192.168.122.3". Object with the conflict has the name "f39"
```
fair enough.  Let's try changing things as well:
```
# cobbler system copy --name f39 --newname f40 --interface default --ip-address 192.168.122.6 --mac-address 52:54:00:2c:39:0c
exception on server: IP address duplicate found "192.168.122.3". Object with the conflict has the name "f39"
```
So is this at all possible to do?  Should one be able to do this?

**Comment by SchoolGuy** [comment]
So I used the copy command in the following way in the past:

```
cobbler system copy --name f39 --newname f40
```

So your impression is correct. I would say that this is a bug then? But we indeed have copy tests both unit (https://github.com/cobbler/cobbler/blob/main/tests/xmlrpcapi/system_test.py#L303-L315) and integration (https://github.com/cobbler/cobbler/blob/main/system-tests/tests/basic-system-rename) but looking at them both tests have no interface.
